src/gui/screen_menu_esp_update.cpp

```diff
@@ -235,7 +235,7 @@ void ScreenMenuESPUpdate::windowEvent(EventLock /*has private ctor*/, window_t *
             log_debug(Network, "ESP read data %ld", readCount);
             if (ferror(opened_file)) {
                 log_error(Network, "ESP flash: Unable to read file %s", current_file->filename);
-                help.SetText(_("Unable to read files. Chek the flash disk and start again"));
+                help.SetText(_("Unable to read files. Try re-flash the last FW and try again"));
```

Comment:
The same....

Reply:
According to conversation the text has been corrected.